test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /w3c
 RUN apt-get update && apt-get install -y git
 RUN git clone --branch level-1 https://github.com/w3c/trace-context.git
 
-FROM mcr.microsoft.com/dotnet/sdk:8.0.410@sha256:b56053d0a8f4627047740941396e76cd9e7a9421c83b1d81b68f10e5019862d7 AS dotnet-sdk-8.0
+FROM mcr.microsoft.com/dotnet/sdk:8.0.411@sha256:e6a5a8d884609907fa8d468b927df765967f6b22f890ce92bd3ae614ca4ae87e AS dotnet-sdk-8.0
 FROM mcr.microsoft.com/dotnet/sdk:9.0.300@sha256:c5b188baf837b0180a14f988815b9cc7a55b836dd6cbe1b6e6523cf3098faaa8 AS dotnet-sdk-9.0
```

Comment:
@martincostello It seems that #6326 correctly prevents the 8.0 -> 9.0 bump, but now it does not correctly bump 9.0.300 to 9.0.301.

Reply:
I guess it's some sort of Dependabot limitation when there's multiple `FROM` statements in a file/next to each other?